.air.toml

```diff
@@ -6,6 +6,7 @@ tmp_dir = "tmp"
   args_bin = []
   bin = "./tmp/main"
   cmd = "go build -o ./tmp/main ."
+  #cmd = "dlv --listen=:40000 --headless=true --api-version=2 --accept-multiclient exec ./tmp/main"
```

Comment:
why commented ?

Reply:
To be able to debug it with an external client, we need to launch the executable through delve. So to enter the delve debug mode, you have to uncomment the line specifying the delve command to run.

Reply:
ok, thx for explanation !